src/components/Accordion/Accordion.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+
+import Accordion from './Accordion';
+
+import type { TAccordionProps } from './Accordion.types';
+
+beforeEach(() => {
+  window.scrollTo = jest.fn();
+});
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+const makeSut = ({
+  header = <div>header</div>,
+  content = <div>content</div>,
+  isOpen = false,
+  ...props
+}: Partial<TAccordionProps>) => {
+  return render(
+    <Accordion isOpen={isOpen} header={header} content={content} {...props} />
+  );
+};
+
+describe('Accordion', () => {
+  it('not render body when [isOpen] is false', () => {
+    makeSut({ isOpen: false });
+
+    const content = screen.queryByText('content');
+
+    expect(content).not.toBeInTheDocument();
+  });
+
+  it('render body when [isOpen] is true', () => {
+    makeSut({ isOpen: true });
+
+    const content = screen.getByText('content');
+
+    expect(content).toBeVisible();
+  });
+});
```

Comment:
A gente evita com que o its estejam muito grandes, pode ajustar? Ai acabamos colocando um describe.

```suggestion
describe('Accordion', () => {
  describe('when isOpen is false', () => {
    it('body cannot render', () => {
      makeSut({ isOpen: false });

      const content = screen.queryByText('content');

      expect(content).not.toBeInTheDocument();
    });
  });

describe('when isOpen is true', () => {
  it('body can render', () => {
    makeSut({ isOpen: true });

    const content = screen.getByText('content');

    expect(content).toBeVisible();
  });


});
});
```